### ISSUE TITLE
Snippet color

### ISSUE THREAD

**Issue by igorepst** [denisidoro/navi#370]
I know the issue with colors in light themed terminals was discussed before and you proposed to change `FZF` colors. However, looking at the code at `display/terminal.rs`, the statements 
```
const COMMENT_COLOR: color::LightCyan = color::LightCyan;
const TAG_COLOR: color::Blue = color::Blue;
const SNIPPET_COLOR: color::White = color::White;
```
actually mean that the snippet will always be printed in white color, am I right? If so, no light background for `FZF` may be used.

**Comment by denisidoro** [comment]
You're right. Sort of.

Using FZF variables may solve your issue in the sense that you can control background colors so that content becomes visible even for light terminal. Examples:

![Screen Shot 2020-08-03 at 14 47 13](https://user-images.githubusercontent.com/3226564/89211605-81581480-d598-11ea-8481-24f1bc838519.png)

![Screen Shot 2020-08-03 at 14 47 29](https://user-images.githubusercontent.com/3226564/89211611-82894180-d598-11ea-81b0-047f96afb639.png)

However, the text color for each column is indeed controlled by these constants and, as of now, can't be changed without recompiling navi.

I guess the ideal solution would be to make this customizable via env variables or something in `~/.config/navi/navirc`. However, I can't promise when I'll implement it, because unfortunately I haven't been able to give as much love to this project as I'd like.

I would gladly accept a PR implementing this, though! 👍

**Comment by denisidoro** [comment]
Related: https://github.com/denisidoro/navi/issues/202

**Comment by igorepst** [comment]
Yes, I learned already today that the project was rewritten in Rust not so long ago. Beforehand I was very confused regarding the seemingly existing workarounds with FZF colors and `--col-colors`

**Comment by igorepst** [comment]
Closed this as a duplicate of #202
